### PR TITLE
format with black, add annotations, return named-tuple

### PR DESCRIPTION
**Pull request by rsokl**

### CODE REVIEW

File: src/facenet_models/__init__.py [hunk 1]
```diff
@@ -1,29 +1,43 @@
...
+
+class _Detections(NamedTuple):
+    boxes: np.ndarray
+    probabilities: np.ndarray
+    face_landmarks: np.ndarray
+
 
 class FacenetModel:
-    def __init__(self, device=None):
+    def __init__(self, device: Optional[str, torch.device] = None):
```
Comment (davidmascharka) [inline src/facenet_models/__init__.py:18]:
```suggestion
    def __init__(self, device: Optional[Union[str, torch.device]] = None):
```

File: src/facenet_models/__init__.py [hunk 2]
```diff
@@ -1,29 +1,43 @@
+from typing import NamedTuple, Optional
```
Comment (davidmascharka) [inline src/facenet_models/__init__.py:1]: and then
```suggestion
from typing import NamedTuple, Optional, Union
```

File: src/facenet_models/__init__.py (hunk 1 above)
Comment (davidmascharka) [reply src/facenet_models/__init__.py:18]: but should this actually just be
```suggestion
    def __init__(self, device: Optional[str] = None):
```
? below we're checking if it's `in {"cuda", "cpu"}`

Comment (rsokl) [reply src/facenet_models/__init__.py]: Ah you are right

**Comment by rsokl** [comment]
Good catch on the type inconsistency. Also.. how did PyCharm not yell at me about the missing `Union`. Anyway, `Optional[str]` should to the trick. Let me know if this looks good

**Pull request merged**